src/System.Runtime.WindowsRuntime/src/System/Windows/Size.cs

```diff
@@ -54,7 +53,7 @@ public double Width
             set
             {
                 if (value < 0)
-                    throw new ArgumentOutOfRangeException(SR.ArgumentOutOfRange_NeedNonNegNum, nameof(Width));
```

Comment:
Let's leave cases like this (changing the property name to value) as they were.  Changing the order is fine, but the text should remain as it was.